cmd/deletefailed.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"sync"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// deletefailedCmd represents the deletefailed command
+var Deletefailed = &cobra.Command{
+	Use:   "deletefailed",
+	Short: "Delete the non running Pods",
+	Run: func(cmd *cobra.Command, args []string) {
+		p := getPlatform(cmd)
+		client, err := p.GetClientset()
+		if err != nil {
+			p.Fatalf("unable to get clientset: %v", err)
+		}
+
+		// gather the list of Pods from all
+		pods, err := client.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{})
```

Comment:
If the intention is to search across all namespaces it's clearer to use `metav1.NamespaceAll` rather than `""`.  From a functionality point of view, I'd prefer the option to specify a namespace to run this commands against, possibly with a check that sets `namespace` to `NamespaceAll` if no argument is given.